django-verdant/rca/templates/rca/programme_page.html

```diff
@@ -127,6 +127,10 @@ <h3>Contact</h3>
                     {% endif %}
                 </div>
 
+                <div class="programme-details row row-flush">
```

Comment:
How does this look if there is no embed? Do we want to hide the div if there is no embed?

Reply:
Good catch.